database/query/profiles.sql

```diff
@@ -6,14 +6,16 @@ INSERT INTO profiles (
     alert,
     name,
     provider_id,
-    subscription_id
-) VALUES ($1, $2, $3, $4, $5, sqlc.arg(provider_id), sqlc.narg(subscription_id)) RETURNING *;
+    subscription_id,
+    display_name
+) VALUES ($1, $2, $3, $4, $5, sqlc.arg(provider_id), sqlc.narg(subscription_id), sqlc.arg(display_name)) RETURNING *;
 
 -- name: UpdateProfile :one
 UPDATE profiles SET
     remediate = $3,
     alert = $4,
-    updated_at = NOW()
+    updated_at = NOW(),
+    display_name = sqlc.arg(display_name)
```

Comment:
This works, I was just wondering if it would get rid of some repeated code if we defaulted to the name for setting and retrieving the display name directly in DB?

Reply:
I actually gave that a try, but didn't figure out how to get postgres to default one column from another. My SQL foo failed me :(